store/badger_test.go

```diff
@@ -0,0 +1,74 @@
+package store
+
+import (
+	"testing"
+
+	"github.com/dgraph-io/badger"
+)
+
+func TestGet(t *testing.T) {
+	dalcKV := NewInMemoryKVStore()
+
+	tc := []struct {
+		name string
+		key  []byte
+		err  error
+	}{
+		{"empty key", []byte{}, badger.ErrEmptyKey},
+		{"not found key", []byte("missing key"), ErrKeyNotFound},
+	}
+
+	for _, tt := range tc {
+		t.Run(tt.name, func(t *testing.T) {
+			_, err := dalcKV.Get(tt.key)
+			if err.Error() != tt.err.Error() {
```

Comment:
Why not `error.Is`?

Reply:
> It's caused by incorrect import. Try again with `github.com/dgraph-io/badger/v3`.

This is what I'm using right now:

![image](https://user-images.githubusercontent.com/3502075/136780120-e52ae1a5-477f-41ba-82c7-58a99ae70c4a.png)
![image](https://user-images.githubusercontent.com/3502075/136780138-c7e09f16-c8a1-439e-a09b-268c7d6d08aa.png)